docs/process.rst

```diff
@@ -343,7 +339,7 @@ Functional test the GUI in two EC2 environments:
     suite, fix the errors before proceeding. If it ends up not being a trivial
     fix, stop this process and create a critical bug/card.
   - If everything went well, publish the new precise and trusty releases by
-    pushing the branch to the respective locations:
+    pushing the ``trusty-release`` branch to the respective locations:
```

Comment:
For some reason this section of bullet points is rendering as a blockquote - could that be fixed as a drive-by?